### PR TITLE
Fix missing list of changes and add a list of messages to notifications

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: integration-tests/tests/models/single.spec.ts [hunk 1]
```diff
@@ -68,6 +68,49 @@ describe('publish', () => {
       expect: 'ignored',
     });
   });
+
+  test.concurrent('CLI output', async () => {
```
Comment (B2o5T) [inline integration-tests/tests/models/single.spec.ts:72]:
```suggestion
  test.concurrent('CLI output', async ({ expect }) => {
```

I guess when using `test.concurrent` you need to destructure `expect`

Comment (kamilkisiela) [reply integration-tests/tests/models/single.spec.ts:72]: none of our `test.concurrent` tests gets `expect` like this

Comment (B2o5T) [reply integration-tests/tests/models/single.spec.ts]: oops, you need to destructure only if you use `toMatchInlineSnapshot` inside `test.concurrent` in vitest

Comment (kamilkisiela) [reply integration-tests/tests/models/single.spec.ts]: :)

**Pull request merged**